components/Account/Memberships/Manage.js

```diff
@@ -154,8 +154,16 @@ const ManageActions = compose(
 
 const Manage = ({ t, membership, highlighted, prolong, waitingMemberships, title, compact, actions }) => {
   const createdAt = new Date(membership.createdAt)
-  const latestPeriod = membership.periods[0]
-  const formattedEndDate = latestPeriod && dayFormat(new Date(latestPeriod.endDate))
+  const latestPeriod = membership.periods
+    .reduce((acc, period) => {
+      return acc && new Date(period.endDate) < new Date(acc.endDate)
+        ? acc
+        : period
+    })
+
+  const latestPeriodEndDate = latestPeriod && new Date(latestPeriod.endDate)
+  const formattedEndDate = latestPeriod && dayFormat(latestPeriodEndDate)
+  const overdue = latestPeriod && latestPeriodEndDate < new Date()
```

Comment:
I'd prefer to keep this logic on the server: [crowdfundings/graphql/resolvers/Membership.js](https://github.com/orbiting/backends/blob/28346f15c64a00451d7668dea735b7c5c124344f/servers/republik/modules/crowdfundings/graphql/resolvers/Membership.js#L9). Maybe it can be simplified there, in the monthly case it is nice to ensure that a charge attempt was made.